MobileBuy/buy/src/androidTest/java/com/shopify/buy/service/BuyTest.java

```diff
@@ -593,6 +593,48 @@ public void failure(BuyClientError error) {
         latch.await();
     }
 
+    @Test
+    public void testGetProductTags() throws Exception {
+        final CountDownLatch latch = new CountDownLatch(1);
+        buyClient.getProductTags(1, new Callback<List<String>>() {
+            @Override
+            public void success(List<String> response) {
+                assertNotNull(response);
+                latch.countDown();
+                // well we can't really test response for some values, as we can't control from client list of tags
```

Comment:
We control the test shop - why can't we check that we got the correct tags here?